packages/fxa-payments-server/src/routes/Checkout/en.ftl

```diff
@@ -1,6 +1,6 @@
 ## Routes - Checkout - New user
 
-new-user-step-1 = 1. Create a { -brand-name-firefox } account
+new-user-step-1b = 1. Create a { -brand-firefox } account
```

Comment:
I think this okay. L10N might prefer a different way of postfixing. We will see!

Reply:
```suggestion
new-user-step-1-v2 = 1. Create a { -brand-firefox } account
```

This seems to be the most common convention used by FxA